src/components/policy/policy_external/test/json/sdl_preloaded_pt.json

```diff
@@ -1,512 +1,1358 @@
-    {
+{
```

Comment:
@ychernysheva I propose to update the `src/components/policy/policy_external/test/CMakeLists.txt` that it copies the `sdl_preloaded_pt.json` file from `src/appMain/` to this directory instead of keeping the copy of the file here.